03.api.md

```diff
@@ -1099,3 +1098,228 @@ handler (const jerry_api_object_t * function_obj_p,
 - [jerry_api_is_function](#jerryapiisfunction)
 - [jerry_api_call_function](#jerryapicallfunction)
 - [jerry_api_release_object](#jerryapireleaseobject)
+
+# jerry_api_create_array_object
+
+**Summary**
+Create new Javascript array object.
```

Comment:
Other parts in the document uses the 'JavaScript' casing, we should follow that.